pkgs/development/node-packages/default.nix

```
@@ -160,6 +160,14 @@ let
 
     node2nix = super.node2nix.override {
       buildInputs = [ pkgs.makeWrapper ];
+      # We need to apply a patch to the source, but buildNodePackage doesn't allow patches.
+      # So we pin the patched commit instead. The commit actually contains two other newer commits
+      # since the last (1.9.0) release, but actually this is a good thing since one of them is a
+      # Hydra-specific fix.
+      src = builtins.fetchTarball {
+        url = "https://github.com/svanderburg/node2nix/tarball/b54d45207427ff46e90f16f2f32771fdc8bff5a4";
+        sha256 = "0x3papb9b8zq66rcviry4zjj0li11maj5rrkh321c87jny2vd5nm";
+      };
```

```suggestion
      src = applyPatches {
        src = fetchFromGitHub {
          owner = "svanderburg";
          repo = "node2nix";
          rev = "node2nix-1.9.0";
          sha256 = "...";
        };
        
        patches = [
          # remove node_ name prefix
          (fetchpatch {
            url = "https://github.com/svanderburg/node2nix/commit/b54d45207427ff46e90f16f2f32771fdc8bff5a4.patch";
            sha256 = "...";
          })
          # set meta platform
          (fetchpatch {
            url = "https://github.com/svanderburg/node2nix/commit/58736093161f2d237c17e75a96529b018cd0ac64.patch";
            sha256 = "...";
          })
        ];
      };
```
let do it like this.